manifest-build-tools/application/release_debian_packages.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+# Copyright 2016, EMC, Inc.
+"""
+This is a command line program that upload a rackhd release to bintray.
+
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/release_debian_packages.py --build-directory b/ \
+--bintray-credential BINTRAY \
+--bintray-subject rackhd \
+--bintray-repo debian \
+--bintray-component main \
+--bintray-distribution trusty \
+--bintray-architecture amd64 \
+--debian-depth 3
+
+The required parameters:
+build-directory: A directory where all the repositories are cloned to. 
+bintray-credential: The environment variable name of bintray credential.
+                    For example: BINTRAY_CREDS=username:api_key
+bintray-subject: The Bintray subject, which is either a user or an organization
+bintray-repo: The Bintary repository name
+
+The optional parameters:
+component: The uploaded component of package, default: main
+distribution: The uploaded distribution of package, default: trusty
+architecture: The uploaded architecture of package, default: amd64
+debian-depth: The depth in top level directory that you want this program look into to find debians.
+"""
+import argparse
+import os
+import sys
+
+try:
+    import common
+except ImportError as import_err:
+    print import_err
+    sys.exit(1)
+
+push_exe_script = "pushToBintray.sh"
```

Comment:
how can your script find it ? it's in a different level of folder: on-tools/manifest-build-tools/
while you are in  on-tools/manifest-build-tools/application/